lxc/file.go

```diff
@@ -912,7 +912,7 @@ func (c *cmdFileMount) Command() *cobra.Command {
 		`Mount files from instances`))
 	cmd.Example = cli.FormatSection("", i18n.G(
 		`lxc file mount foo/root fooroot
-   To mount /root from the instance and onto the local fooroot directory.`))
+   To mount /root from the instance foo and onto the local fooroot directory.`))
```

Comment:
Please can you remove the "and" after "foo"? Thanks

Reply:
OK!!

Reply:
Oops, I forgot make i18n ... wait a moment.